mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -0,0 +1,238 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ml.feature
+
+import org.apache.spark.annotation.AlphaComponent
+import org.apache.spark.ml.{Estimator, Model}
+import org.apache.spark.ml.param.{HasInputCol, ParamMap, Params, _}
+import org.apache.spark.mllib.feature
+import org.apache.spark.mllib.linalg.{Vector, VectorUDT}
+import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types._
+import org.apache.spark.util.Utils
+
+/**
+ * Params for [[Word2Vec]] and [[Word2VecModel]].
+ */
+private[feature] trait Word2VecParams extends Params
+  with HasInputCol with HasMaxIter with HasLearningRate {
+
+  /**
+   * The dimension of the code that you want to transform from words.
+   */
+  val vectorSize = new IntParam(
+    this, "vectorSize", "the dimension of codes after transforming from words", Some(100))
+
+  /** @group getParam */
+  def getVectorSize: Int = get(vectorSize)
+
+  /**
+   * Number of partitions for sentences of words.
+   */
+  val numPartitions = new IntParam(
+    this, "numPartitions", "number of partitions for sentences of words", Some(1))
+
+  /** @group getParam */
+  def getNumPartitions: Int = get(numPartitions)
+
+  /**
+   * A random seed to random an initial vector.
+   */
+  val seed = new LongParam(
+    this, "seed", "a random seed to random an initial vector", Some(Utils.random.nextLong()))
+
+  /** @group getParam */
+  def getSeed: Long = get(seed)
+
+  /**
+   * The minimum count of words that can be kept in training set.
```

Comment:
this wording is unclear, perhaps it would just be easier to copy the comments from the implementation?

so for example: "The minimum number of times a token must appear to be included in the word2vec model's vocabulary"

https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala